### ISSUE TITLE
NMM 62.13, files/installed mods not visible in Mods tab and cannot be added visibly again for installation/overwriting

### ISSUE THREAD

**Issue by h-lucz** [Nexus-Mods/NMM-Test-Builds#91]
For your info.
Texture packs for Worsin's Immersive Power Armor Garage (loose files no esp, esm).
Archive names:
WIPAG -Style-3- Solid Colors and Black-v1.7-2249-1-7.7z
WIPAG -Style-5- Colors and Colored Flames-v1.7-2249-1-7.7z

Both files actually had been installed with NMM before the NMM 62+ test run, so they are unpacked in the FO 4 Data directory where they are supposed to be. They were downloaded from Nexus (manually).
Currently the files are not listed in Mods tab.
Trying to add them again for installation (overwriting) does not add them visibly to the Mods tab (should be usually in grey), although the download manager indicates them as "Worsin's Immersive Power Armor Garage (WIPAG) -  has been added"
Trying to add the files using the shell extension feature does not help either.
The online profile indicates these files as missing/not being on Nexus.
Files seem actually to work in the game - like manually unpacked, copied and pasted - what I actually have not done.
At least FYI.

**Comment by DuskDweller** [comment]
Did you use the "Get missing download id" function using the latest test release?
I have no issues with those files using the latest build.

**Comment by h-lucz** [comment]
Thanks for your reply. I ran the get missing/outdated Id and the Gets missing mod info. The latter requires highlighting a mod first. In the Gets missing mod info menu all styles and the main file are listed. Clicking the missing one style 3 or 5 replaces the previously higlighted one in Mods tab. In the NMM mods folder and the virtual install folder the archives main, 2, 3, 4, and 6 are listed, but style 1 is missing. All texture files for the styles 1 - 6 are in the FO 4 Data, textures, actor, power armore etc.
Strange, isn't it.

**Comment by DuskDweller** [comment]
Are you sure you didn't install Style 1 manually? Cause NMM can't manage loose (manually installed) files, even if they're coming from a nexus mod file.

**Comment by h-lucz** [comment]
Thank you DuskDweller, I have not installed files manually since I ran FOMM and later NMM. That is for sure. In the image, NMM Mods tab you will see Style 1, because it replaced the previous Style 3 or 5 when running the Gets mods info. Style 3 and 5 are not added visibly (althought the download manager says " .... has been added"). Although Style 1 is displayed in Mods tab, it is not listed in the NMM mods folder (image). All the WIPAG files are in the FO 4  Data-/and subfolders as there were installed at the end of January this year. All files seem to work in game as the styles 1 to 6 are available in the PA workshop. I will add 4 images as they can say more than words. Maybe it is on my end only, as there are no confirming reports so far. 
The main file fomod 1.7.01c indeed is not listed any more on Worsin's mod page.
Just for info some images as they can say more than words. Thanks again.

![gets missing mod info replaces marked mod](https://cloud.githubusercontent.com/assets/18660199/15395346/9fbd3590-1dd7-11e6-9c9e-d15b53ee5fa6.png)
![nmm mods folder wipag](https://cloud.githubusercontent.com/assets/18660199/15395347/9fbeefe8-1dd7-11e6-80c0-7d63ebe900fa.png)
![wipag files data textures](https://cloud.githubusercontent.com/assets/18660199/15395348/9fc19284-1dd7-11e6-915a-9566f88904a5.png)
![worsins mod style 3 and 5 missing](https://cloud.githubusercontent.com/assets/18660199/15395349/9fc46680-1dd7-11e6-93c6-298b7ed59ee0.png)

**Comment by KunoMochi** [comment]
It looks like you have 5 of the mods installed even though 6 of them are shown in the Mods folder.

Can you share the uploaded profile with the missing mods?

Also, can you right-click on each of the mods in NMM and show us what the file name is for them?
![nmm_contextmenu](https://cloud.githubusercontent.com/assets/7691651/15411792/76f2d070-1df0-11e6-9718-03f7cec7cc1e.png)

**Comment by DuskDweller** [comment]
I'm unable to replicate this issue, try this:
- close NMM
- **move** those mod archives on your desktop
- go in the Mods/cache folder and **remove** the folders for those archives
- launch NMM
- drag&drop those mod archives from your desktop to the NMM Mods tab
- on the left bar click on the binoculars icon and select "fix missing or outdated download ids".

Let me know if it works :)

**Comment by h-lucz** [comment]
@Kuno,
I can upload the profile. The files are indicated as missing, also files that are in mods tab. Will try DuskDweller advice.

**Comment by h-lucz** [comment]
@DuskDweller and @KunoMochi , DuskDweller's hint: Drag&Drop did not work, but after trying to download Style 3 from the mod page with NMM, Mods tab unexpectedly displayed Style 3 and 5 as not installed (slashed red circle). Could be installed with select and clicking the green checkmark to install. Style 1 still in Data and Mods tab, but not in NMM Mod Cache (different file name generally in comparison with Style 2 ff). The files are in the Data folder. I'll try to modify the style 1 name equivalent to the Style 2 ff files. We'll see. Ah, Style 3 and 5 are now in Mods tab and have a DL ID. Can't expand on more right away - online profile update ect. as Nexus seems to be not available again ... See you, Thank you Dusk.

**Comment by h-lucz** [comment]
Modifying Style 1 to the archive names of style 2 ff worked. Displayed in mods tab and in NMM mods and cache folder. Still misleading info on online profiles after update and upload with new profile name.
Mod ID in Mods tab now clear.
I'm going to change the file names in Mods tab equivalent to the new folder names in NMM Mod Cache as they match those on the mod page while names in Mods tab do not. Your advice to regenerate files and make NMM access them in Mods tab worked as described in my previous post.
Just an image to display that the new re-generated Style file names do not match the online names. Think it is due to that, that the online profile indicates them as missing. Will let you know the result of course. Thanks for your help.

![wipag online missing after cleaning nmm](https://cloud.githubusercontent.com/assets/18660199/15448075/a0d5fcd4-1f56-11e6-9460-6888c3b65368.png)

**Comment by h-lucz** [comment]
Just for info. I renamed the WIPAG file names in mods tab equivalent to the downloaded archive names (identical with the names in NMM Mods and Cache folder. I created a new profile an uploaded the test file. Except style 1 all other files are missing online. Image:

![file names adopted](https://cloud.githubusercontent.com/assets/18660199/15448821/d677d1b2-1f6c-11e6-8207-8a991dc22d20.png)

Running the Check/fix for missing/outdated download IDs changes the files again to the originals after installation. The altered names are not detected either. See previous report.

Most important for me is the fact, that your advice worked to make NMM access the addition of WIPAG files visible and for installation :-).
Thank you. I still assume that all is on my end and not on NMM. For Skyrim no issues after renaming customized esp archives exactly as the authors' archive patch packs concerning the online recognition (only install and download for users might cause issues due to the authors original archive).
Thanks again.
